Lib/fontTools/feaLib/ast.py

```diff
@@ -210,21 +220,24 @@ def add_class(self, gc):
 
 
 class GlyphClassName(Expression):
-    """A glyph class name, such as @FRENCH_MARKS."""
+    """A glyph class name, such as ``@FRENCH_MARKS``. This must be instantiated
+    with a :class:`GlyphClassDefinition` object."""
     def __init__(self, glyphclass, location=None):
         Expression.__init__(self, location)
         assert isinstance(glyphclass, GlyphClassDefinition)
```

Comment:
Assertions can be disabled, so they shouldn’t be used to catch violations of documented type requirements. It would be better to use something like:
```python3
        if not isinstance(glyphclass, GlyphClassDefinition):
            raise TypeError
```

Reply:
The `assert` isn't relevant to this documentation PR.